<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KCurrencyEditDlg</class>
 <widget class="QDialog" name="KCurrencyEditDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>516</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string comment="@title:window">Currencies</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTreeWidget" name="m_currencyList">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>1</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::CustomContextMenu</enum>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::ExtendedSelection</enum>
     </property>
     <property name="rootIsDecorated">
      <bool>false</bool>
     </property>
     <property name="allColumnsShowFocus">
      <bool>true</bool>
     </property>
     <column>
      <property name="text">
       <string comment="@title:column currency name">Name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string comment="@title:column currency ISO4217 code">ISO 4217 ID</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string comment="@title:column currency symbol">Symbol</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="2">
      <widget class="QPushButton" name="m_removeCurrencyButton">
       <property name="text">
        <string comment="@action:button Remove currency">Remove</string>
       </property>
       <property name="icon">
        <iconset theme="list-remove">
         <normaloff>.</normaloff>.</iconset>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="m_addCurrencyButton">
       <property name="text">
        <string comment="@action:button Add currency from existing list">Add</string>
       </property>
       <property name="icon">
        <iconset theme="list-add">
         <normaloff>.</normaloff>.</iconset>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QPushButton" name="m_editCurrencyButton">
       <property name="text">
        <string comment="@action:button Edit currency">Edit</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="4">
      <widget class="QPushButton" name="m_removeUnusedCurrencyButton">
       <property name="text">
        <string comment="@action:button">Remove unused currencies</string>
       </property>
       <property name="icon">
        <iconset theme="edit-clear">
         <normaloff>.</normaloff>.</iconset>
       </property>
      </widget>
     </item>
     <item row="4" column="3">
      <widget class="QPushButton" name="m_closeButton">
       <property name="text">
        <string comment="@action:button Close currency dialog">Close</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0" colspan="3">
      <widget class="QPushButton" name="m_selectBaseCurrencyButton">
       <property name="text">
        <string comment="@action:button">Select as base currency</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="m_newCurrencyButton">
       <property name="text">
        <string comment="@action:button Create new currency">New</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>m_currencyList</tabstop>
  <tabstop>m_addCurrencyButton</tabstop>
  <tabstop>m_newCurrencyButton</tabstop>
  <tabstop>m_removeCurrencyButton</tabstop>
  <tabstop>m_editCurrencyButton</tabstop>
  <tabstop>m_removeUnusedCurrencyButton</tabstop>
  <tabstop>m_selectBaseCurrencyButton</tabstop>
  <tabstop>m_closeButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>m_closeButton</sender>
   <signal>clicked()</signal>
   <receiver>KCurrencyEditDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>296</x>
     <y>488</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>257</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
